third_party/gpus/cuda_configure.bzl

```diff
@@ -646,15 +699,31 @@ def _find_cudnn_header_dir(repository_ctx, cudnn_install_basedir):
   Returns:
     The path of the directory containing the cudnn header.
   """
-  if repository_ctx.path(cudnn_install_basedir + "/cudnn.h").exists:
-    return cudnn_install_basedir
-  if repository_ctx.path(cudnn_install_basedir + "/include/cudnn.h").exists:
-    return cudnn_install_basedir + "/include"
+  for relative_path in CUDA_INCLUDE_PATHS:
+    if repository_ctx.path("%s/%scudnn.h" % (cudnn_install_basedir, relative_path)).exists:
+        return ("%s/%s" % (cudnn_install_basedir, relative_path))[:-1]
   if repository_ctx.path("/usr/include/cudnn.h").exists:
```

Comment:
This is a bit of a wrinkle. The alternatives I could think of are:

1) Leave as-is -- not worth adding complexity for a single special case
2) Add CUDA_INCLUDE_ABSOLUTE_PATHS
3) Support leading "/" in paths and add logic for handling them. At that point the per-function logic gets complex enough that I'd want to extract helpers.

WDYT?